src/libpisp/backend/tiling/crop_stage.cpp

```diff
@@ -98,7 +121,7 @@ void CropStage::PushCropDown(Interval interval, Dir dir)
 	downstream_->PushCropDown(output_interval_, dir);
 }
 
-void CropStage::Reset()
+bool CropStage::BranchInactive() const
 {
-	started_ = false;
+	return !output_interval_.length;
```

Comment:
I guess I feel slightly weird that `BranchInactive()` looks at the output_interval, but `BranchComplete()` doesn't - it slight offends my need for symmetry. Should `BranchCompletel()` look at `output_interval_.End()` or something? Or do I worry too much?